service/storage_proxy.hh

```diff
@@ -449,7 +449,8 @@ public:
             scheduling_group_key stats_key, gms::feature_service& feat, const locator::shared_token_metadata& stm, locator::effective_replication_map_factory& erm_factory, netw::messaging_service& ms);
     ~storage_proxy();
 
-    remote& remote();
+    const struct remote& remote() const;
+    struct remote& remote();
```

Comment:
nit: Is it truly necessary to emphasize `remote&` as `struct remote&`

Reply:
Yes, because otherwise it doesn't compile

Reply:
Smth about conflicting `remote` as struct and `remove` as method names?

Reply:
> Smth about conflicting remote as struct and remote as method names?

yes